.gitignore

```diff
@@ -10,13 +12,6 @@ Cargo.lock
 .cargo/config
 .cargo/config.toml
 
-# IDE files
-.idea
-.vscode
-.zed
-dxcompiler.dll
```

Comment:
The dx ignore should still be kept. They aren't IDE files, they are pretty specific to using bevy with a DX12 backend so I think it make sense to keep them[